docs/platforms/javascript/guides/solidstart/features/solid-router.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Solid Router
+description: "Learn about Sentry's Solid Router integration."
+---
+
+<Note>
+  The routing instrumentation supports [Solid Router](https://docs.solidjs.com/solid-router) 0.13.4 and up.
+</Note>
+
+The Sentry SDK provides a routing instrumentation for Solid Router to create navigation spans to ensure
+you collect meaningful performance data about the health of your page loads and associated requests.
+
+To get started, import `solidRouterBrowserTracingIntegration` from `@sentry/solidstart/solidrouter` and add it to `Sentry.init`
+instead of the regular `Sentry.browserTracingIntegration` to enable performance tracing.
```

Comment:
Just a general question: Isn't the `browserTracingIntegration` always included? Not sure if there was a PR for solidstart as well.

Reply:
It's not yet, I should create an issue for that.

Reply:
https://github.com/getsentry/sentry-javascript/issues/13352